rollbar/__init__.py

```diff
@@ -626,6 +626,9 @@ def _report_exc_info(exc_info, request, extra_data, payload_data, level=None):
         data['body'] = {
             'trace_chain': trace_chain
         }
+        if payload_data and payload_data['body'] and payload_data['body']['trace']:
+            payload_data['body']['trace_chain'] = payload_data['body']['trace']
```

Comment:
This looks incorrect - `trace_chain` should be a list of `trace`s

Reply:
Yes yes, d'oh

Reply:
What if we added it to the custom data instead of trying to merge it into a trace?